crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -865,13 +803,25 @@ impl Stage for SystemStage {
                 // Run systems that want to be at the start of stage.
                 for container in &mut self.exclusive_at_start {
                     if should_run(container, &self.run_criteria, default_should_run) {
-                        #[cfg(feature = "trace")]
-                        let _system_span = bevy_utils::tracing::info_span!(
-                            "exclusive_system",
-                            name = &*container.name()
-                        )
-                        .entered();
-                        container.system_mut().run(world);
+                        {
+                            #[cfg(feature = "trace")]
+                            let _span = bevy_utils::tracing::info_span!(
+                                "exclusive_system",
+                                name = &*container.name()
+                            )
+                            .entered();
+                            container.system_mut().run((), world);
+                        }
+
+                        {
+                            #[cfg(feature = "trace")]
```

Comment:
Shouldn't we focus on making as few breaking changes as possible in this PR so we can move forward?

Reply:
It's already a breaking change for people who call `exclusive_system.run(world)`.

Reply:
This code will probably be gone with stageless, so I'm fine with just leaving this as is. Just left a comment since it's a small drop in code quality.